### PR TITLE
docs(icon): add overview file

### PR DESCRIPTION
**Pull request by mmalerba**

### CODE REVIEW

File: src/lib/icon/OVERVIEW.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# md-icon
+
+`md-icon` is a component that displays an icon, which can be a font glyph or an SVG image.
+
```
Comment (jelbourn) [inline src/lib/icon/OVERVIEW.md]: How about
```
`md-icon` makes it easier to use _vector-based_ icons in your app. 
This directive supports both icon fonts and SVG icons, but not bitmap-based formats (png, jpg, etc.).
```

File: src/lib/icon/OVERVIEW.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+`MdIconRegistry`. After an icon set is registered, each of its embedded icons can be accessed by
+their `id` attributes. To display an icon from an icon set, use the `svgIcon` input in the same way
+as for individually registered icons.
+
+Multiple icon sets can be registered in the same namespace. Requesting an icon whose id appears in
+more than one icon set, the icon from the most recently registered set will be used.
+
+Note that all SVG icons are fetched via XmlHttpRequest, and due to the same-origin policy their URLs
+must be on the same domain as the containing page, or their servers must be configured to allow
+cross-domain access.
```
Comment (jelbourn) [inline src/lib/icon/OVERVIEW.md]: We need to note in here that, to guard against XSS, any URLs given to the `MdIconRegistry` must be marked as trusted resource URLs by using Angular's `DomSantizer` service.

**Comment by mmalerba** [comment]
Done

**Comment by jelbourn** [comment]
LGTM

**Pull request merged**